articles/azure/azs-how-setup-monitoring.md

```diff
@@ -0,0 +1,180 @@
+                            "apiVersion": "2015-11-01-preview",
+                            "location": "[parameters('WorkspaceLocation')]",
+                            "name": "[concat('InfrastructureInsights', '(', parameters('WorkspaceName'),')')]",
+                            "type": "Microsoft.OperationsManagement/solutions",
+                            "dependsOn": [
+                                "[concat('Microsoft.OperationalInsights/workspaces/', parameters('WorkspaceName'))]"
+                            ],
+                            "properties": {
+                                "workspaceResourceId": "[resourceId('Microsoft.OperationalInsights/workspaces/', parameters('WorkspaceName'))]"
+                            },
+                            "plan": {
+                                "name": "[concat('InfrastructureInsights', '(', parameters('WorkspaceName'),')')]",
+                                "publisher": "Microsoft",
+                                "product": "[Concat('OMSGallery/', 'InfrastructureInsights')]",
+                                "promotionCode": ""
+                            }
+                        }
+                    ]
+                }
+            ]
+        }
+    </code></pre>
+
+    - Click **Save**
+
+    - Select a **Resource Group** or create a new one.
+
+    - Select a **Location**
+
+    - Under the *Settings* heading. Fill in the parameters `Workspace Name` and `Workspace Location`.
+
+    - Click `I agree to the terms and conditions stated above`.
+
+    - Click **Purchase**.
+
+    - The *Log Analytics Workspace* and required solutions will begin deploying.
+
+3. Once deployment is complete, navigate to the *resource group* you placed the **Log Analytics workspace** within.
+
+    - Click the newly create workspace. On the *new* blade, under settings, select *Advanced settings*, then select *Windows* or *Linux* servers depending on the *VM Type* you wish to add analytics too.
+    Note down the **Workspace ID** and **Primary Key** values.
+
+        ![Log Analytics workspace advanced settings](images\azs-browser-log-analytics-workspace-advanced-settings.PNG)
+
+    - Within *Advanced settings*, Select the **Data** blade, and click *`VM Type` Performance Counters.*
+
+    - Ensure all *counters* are selected and click **Add the selected performance counters**.
+
+    - Click **Save**.
+
+        ![Performance Counters](images/azs-browser-example-performance-counters.PNG)
+
+        > [!NOTE]
+        > For Linux Performance Counters, ensure **Apply below configuration to my machines** is selected.
+
+4. Login to the Azure Stack portal:
+
+    <https://portal.frn00006.azure.ukcloud.com>
+
+5. Navigate to the VM which you wish to enable **Azure Monitor** on and under settings, select the *extensions* blade.
+
+    > [!WARNING]
+    > For any monitoring to work correctly. The VM **must** have HTTPS (Port 443) enabled in the **Network Security Group** rules.
```

Comment:
comma instead of full stop:
For any monitoring to work correctly, the VM must ...